projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_configuration_plugin.py

```diff
@@ -29,6 +29,18 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             default_value=None,
             description="Default Ingestion target to be used.",
         )
+        config_builder.add(
+            key="INGEST_PLUGIN_EXECUTION_SEQUENCE",
```

Comment:
What would happen if I've set both this parameter and INGEST_METHOD_DEFAULT?

Reply:
I totally agree with @gageorgiev. @doks5 could you please add examples and use cases. I am not sure what problem we are trying to solve.

Reply:
INGEST_METHOD_DEFAULT would be ignored.